src/main/java/com/metamx/common/StreamUtils.java

```diff
@@ -44,9 +48,12 @@ public static void copyToFileAndClose(InputStream is, File file) throws IOExcept
     finally {
       CloseQuietly.close(is);
     }
+    return FileUtils.FileCopyResult.buildResult(file.toPath());
```

Comment:
Reverted, but still made it return `long` since most copy functions anywhere return at least the size of what was copied.